README.md

```diff
@@ -16,11 +16,14 @@ Gothenburg University is currently buying further development and service from p
 
 Before you decide on this, the Faculty of Science at University of Gothenburg has included Scan-o-matic among its high-throughput phenomics infrastructure and it is our expressed interest that external researchers come to us. If you are interested there's some more information and contact information here: [The center for large scale cell based screeening](http://cmb.gu.se/english/research/microbiology/center-for-large-scale-cell-based-screening). It is yet to become listed on the page, but don't worry, it is part of the list.
 
-## Current focus
+# Setting up and running
 
-The ability to add new Cell Count Calibrations via UI is being developed.
-
-Completing and securing up the code for the first release of 2.x
+1. Clone this library `git@github.com:Scan-o-Matic/scanomatic-standalone.git`
+2. And change directory: `cd scanomatic-standalone`
+3. Build the image `docker-compose build`
+4. Create an environment-file or export the `SOM_PROJECTS_ROOT` and `SOM_SETTINGS` variables. Each pointing to a location where you wish to store your projects and settings respectively. See https://docs.docker.com/compose/environment-variables/#the-env-file for specifying them in an env-file, else include them in your `.bashrc`.
```

Comment:
```suggestion
4. Create an environment-file or export the `SOM_PROJECTS_ROOT` and `SOM_SETTINGS` variables. Each pointing to a location where you wish to store your projects and settings respectively. See https://docs.docker.com/compose/environment-variables/#the-env-file for specifying them in an env-file, else include them in e.g. your `.bashrc`.
```

Reply:
maybe (though those who don't use bash probably know that this doesn't apply to them)